app/views/pages/derivatives.html.erb

```diff
@@ -0,0 +1,162 @@
+        <table id="auk_deriv_accord_table">
+          <tbody class="auk_deriv_tbody_table">
+            <tr class="auk_deriv_tr_table">
+              <td class="auk_deriv_td_table"><strong>Tutorial</strong></td>
+              <td class="auk_deriv_td_table">
+                <ul>
+                  <li class="card_li"><strong><%=link_to("Text Analysis Part One: Beyond the Keyword Search: Using AntConc", "/derivatives/text-antconc")%></strong>: This tutorial explores how you can explore text within a web archive using the AntConc tool.</li>
+                  <li class="card_li"><strong><%=link_to("Text Analysis Part Two: Sentiment Analysis With the Natural Language Toolkit", "/derivatives/text-sentiment")%></strong>: This tutorial explores how you can calculate the positivity or negativity (in an emotional sense) of web archive text.</li>
+                </ul>
+              </td>
+            </tr>
+            <tr class="auk_deriv_tr_table">
+              <td class="auk_deriv_td_table"><strong>File Output</strong></td>
+              <td class="auk_deriv_td_table"><kbd>.txt</kbd></td>
+            </tr>
+            <tr class="auk_deriv_tr_table">
+              <td class="auk_deriv_td_table"><strong>Files Generated</strong></td>
+              <td class="auk_deriv_td_table">A <strong>text file</strong> is generated which can be used in various other programs and software.</td>
+            </tr>
+            <tr class="auk_deriv_tr_table">
+              <td class="auk_deriv_td_table"><strong>Additional Resources</strong></td>
+              <td class="auk_deriv_td_table">
+                <p class="about_p">Working with lots of text can be challenging. Fortunately, there are lots of resouces on the web to help teach you the basics of text analysis.</p>
+                <ul>
+                  <li class="card_li"><strong><a href="https://programminghistorian.org">Programming Historian</a></strong>: The Programming Historian has many lessons, including a specific section on <a href="https://programminghistorian.org/en/lessons/?topic=distant-reading">distant reading."</a></li>
+                  <li class="card_li"><strong><a href="http://tapor.ca/home">Text Analysis Portal for Research</a></strong>: TAPoR, or the Text Analysis Portal for Research, provides a curated collection of tools that can be used to explore text.</li>
+                  <li class="card_li"><strong><a href="https://uottawa.libguides.com/DH/textual">Digital Humanities Guide at University of Ottawa</a></strong>: This is a great guide that highlights several approaches to text analysis in the field.</li>
+                  <li class="card_li"><strong><a href="http://voyant-tools.org/">Voyant Tools</a></strong>: This is more of a tool than an introductory resource, but it's a great "gateway drug" into the powers of text analyis.</li>
```

Comment:
I know it was I who wrote "gateway drug," but on sober second thought (heh) maybe we should change

`it's a great "gateway drug" into the powers of text analyis` to

`it's a gentle yet powerful introduction to the powers of text analysis.`